### YamlMime:UniversalReference
ms.openlocfilehash: 165dfce52da9abb9b6fd4a764d0fef0e53b37c9f
ms.sourcegitcommit: 286f52ea38c9eff2ec9d4f8cabeb86f62fd9c406
ms.translationtype: MT
ms.contentlocale: ru-RU
ms.lasthandoff: 08/22/2018
ms.locfileid: "41692113"
items:
- uid: azure-iot-common.InternalServerError
  name: InternalServerError
  fullName: InternalServerError
  children:
  - azure-iot-common.InternalServerError.Error
  - azure-iot-common.InternalServerError.constructor
  langs:
  - typeScript
  type: class
  summary: <span data-ttu-id="59cb9-101">Ошибка возникает, когда произошла внутренняя ошибка сервера.</span><span class="sxs-lookup"><span data-stu-id="59cb9-101">Error thrown when an internal server error occured.</span></span> <span data-ttu-id="59cb9-102">Нашли ошибку?</span><span class="sxs-lookup"><span data-stu-id="59cb9-102">You may have found a bug?</span></span>
  extends:
    name: Error
  package: azure-iot-common
- uid: azure-iot-common.InternalServerError.Error
  name: Error
  fullName: Error
  children: []
  langs:
  - typeScript
  type: property
  summary: ''
  syntax:
    content: 'static Error: ErrorConstructor'
    return:
      type:
      - ErrorConstructor
  package: azure-iot-common
- uid: azure-iot-common.InternalServerError.constructor
  name: InternalServerError
  children: []
  type: constructor
  langs:
  - typeScript
  summary: ''
  syntax:
    content: 'new InternalServerError(message?: string)'
    parameters:
    - id: message
      type:
      - string
      description: ''
      optional: true
  package: azure-iot-common
